tests/behat/features/cu_advanced_design_bundle/blocktitleicons.feature

```diff
@@ -0,0 +1,28 @@
+@AdvDesign
+Feature: Font Awesome Icons in block titles
+In order to add visual interest to my site
+As an authenticated user
+I should be able to attach Font Awesome icons to my block titles
+
+# TESTING NOTE: ATTACHING A BLOCK TO A PAGE IS VERY DIFFICULT WITH BEHAT; 
+# IT CAN BE DONE BUT THE TEST IS NOT ROBUST; SO IT HAS TO BE A FUNCTIONAL TEST, NOT AN AUTOMATED ONE
```

Comment:
To clarify, pretty much any Behat test is a functional test as compared with a unit or integration test. They are all automated in this repo so they can run on Travis. The tests themselves are agnostic to whether they run manually or in an automated fashion.

Reply:
By 'functional' I meant 'not automated' as in 'a human has to do this'